java/src/jmri/jmrit/picker/PickPanel.java

```diff
@@ -38,9 +42,10 @@
     JTextField _userNametext;
     jmri.jmrit.picker.PickFrame _pickTables; // Opened from LogixTableAction
 
-    public PickPanel(PickListModel[] models) {
+    @SuppressWarnings("unchecked") // cast as part of generic array ctor
+    public PickPanel(PickListModel<?>[] models) {
         _tabPane = new JTabbedPane();
-        _models = new PickListModel[models.length];
+        _models = (PickListModel<NamedBean>[]) new PickListModel[models.length]; // generic array construction
```

Comment:
It seems to me that this line should be
`_models = new PickListModel<NamedBean>[models.length]; // generic array construction`
but I might be wrong.

Reply:
@bobjacobsen 
I tried `_models = new PickListModel<NamedBean>[models.length];` but it didn't work.

Instead I tried to convert `_models` to a List and it seems to work fine. See PR #10485 and the commit https://github.com/JMRI/JMRI/pull/10485/commits/886894129f4702db31927b669b4857f7fae29289 .

I haven't used PickList much so I'm not sure on how to properly test it but #10485 has passed `Static Analysis` and `Run Separate Tests`.

Feel free to include my commit here if you want.

Reply:
Java doesn't allow you to create an array of something like PickListModel<NamedBean>.  You have to create an array of the generic type (without the NamedBean) and cast it over.  The problem is remembering the exact syntax for that cast, because I don't do it very often...

I'll take a look at #10485, thanks.

Reply:
Merged #10485 into this PR's branch